src/theme.js

```diff
@@ -30,7 +30,7 @@ const theme = {
     title: "Lato, Quicksand, sans-serif"
   },
   carts: {
-    sizes: ["XS", "S", "M", "L", "XL"],
+    sizes: { regular: ["XS", "S", "M", "L", "XL"], mobile: ["XS", "S", "M"] },
```

Comment:
Should the mobile users get list that looks like S M L? Seems a little odd that you'd get an XS option before getting the S M and L options